src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx

```diff
@@ -121,7 +127,7 @@ export const JobDetailAbout = () => {
         <div className="hidden md:block">{detailJSX}</div>
 
         <Input className="hidden md:block" id="copy-url" value={url} postfix={inputJSX} />
-        {!jobDetail.applied && (
+        {!jobDetail.applied && isUser && (
```

Comment:
isUser should not be here, if user is not logged-in, still he/she should be able to click on apply, then he/she would be redirected to login page